ui/src/ScholarReader.tsx

```diff
@@ -673,7 +628,7 @@ class ScholarReader extends React.PureComponent<Props, State> {
     });
   }
 
-  setFindQuery(findQuery: FindQuery) {
+  setFindQuery = (findQuery: FindQuery): void => {
```

Comment:
I'm fine with all these explicit `void` annotations, though also want to ask if there's a reason that you added them in for my personal edification.

Reply:
It's a practice we follow in the S2 webapp. Makes it crystal clear the intent of this function is not to return anything. More of a code style thing than a functionality thing.